source/Calamari/Commands/FindPackageCommand.cs

```diff
@@ -14,17 +18,23 @@ namespace Calamari.Commands
     public class FindPackageCommand : Command
     {
         readonly ILog log;
+        readonly IVariables variables;
         readonly IPackageStore packageStore;
+        readonly IJournal packageJournal;
         string packageId;
         string rawPackageVersion;
         string packageHash;
         bool exactMatchOnly;
         VersionFormat versionFormat = VersionFormat.Semver;
 
-        public FindPackageCommand(ILog log, ICalamariFileSystem fileSystem, IPackageStore packageStore)
+        //TODO: move journal/variable(?) stuff to mediator
```

Comment:
Ideally, FindPackageCommand wouldn't need to know about the Journal, and we'd just handle a mediator event to trigger recording package use.  For a single command though, I think that's overkill.